src/main/native/com/code_intelligence/jazzer/driver/android_tooling.cpp

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if defined(_ANDROID)
```

Comment:
Assuming this file is only ever useful on Android, why do we need this define? If the file is compiled on other platforms, we could instead try to prevent that.

Reply:
update this to not be included in non-Android builds